content/about/membership.md

```diff
@@ -0,0 +1,45 @@
++++
++++
+
+# Becoming a member
+
+### Why become a member
+
+ - We aspire to represent and support all research software engineers in the
+   Nordics and Baltics. When we represent you and form collaborations, our word
+   has more weight when we have more members.
+
+   For example, we work to get software journals and software publications the
+   acknowledgment they deserve as contributions to science. Having more members
+   means we represent a larger percentage of academic researchers. We can also
+   draw on our members experiences to support our case.
+
+ - We run events that are useful to our members, such as seminars, workshops
+   and an annual unconference. Your membership fee helps make these events
+   possible.
+
+   In-person events may have a participation fee and members get discounts.
+
+ - An association is owned by its members. As a member you can participate in
+   the associations annual meeting and vote on the direction the association
+   is evolving.
+
+
+### How Do I Sign Up?
+
+To become a member you need to
+ - register by filling out the membership form (https://forms.gle/FXdTv2mtBn53gHxx9).
```

Comment:
don't know if it's intentional, but it's not rendered as link

Reply:
Not intentiional, fixed